Documentation/USERGUIDE.md

```diff
@@ -1584,6 +1584,12 @@ The Cordio Bluetooth Low Energy (BLE) library provides a full BLE stack for micr
 - MAX32680
 - MAX32690
 
+#### Cordio Documentation
+
+- [Architecture](ARCHITECTURE.md)
```

Comment:
I think we'll have to add a path for these links to work.

Reply:
tested it a couple different times on my machine, since the workflow copies those file in the same directory, pathing like this works fine